synapse/api/room_versions.py

```diff
@@ -57,16 +57,19 @@ class RoomVersion:
     state_res = attr.ib()  # int; one of the StateResolutionVersions
     enforce_key_validity = attr.ib()  # bool
 
-    # bool: before MSC2261/MSC2432, m.room.aliases had special auth rules and redaction rules
+    # Before MSC2261/MSC2432, m.room.aliases had special auth rules and redaction rules
```

Comment:
MSC2261 wasn't merged, can we update it while we're here? 😄 

```suggestion
    # Before MSC2432, m.room.aliases had special auth rules and redaction rules
```